ckanext/geocat/harvester.py

```diff
@@ -39,13 +42,20 @@ def info(self):
     def validate_config(self, config):
         if not config:
             return config
+
         try:
             config_obj = json.loads(config)
-            if 'delete_missing_datasets' in config_obj:
-                if not isinstance(config_obj['delete_missing_datasets'], bool):
-                    raise ValueError('delete_missing_dataset must be boolean')
         except Exception as e:
-            raise e
+            raise ValueError('{} configuration could not be parsed. An error {} occured'
+                                 .format(e))
+
+        if 'delete_missing_datasets' in config_obj:
+            if not config_obj['delete_missing_datasets'].lower() in ['true', 'false']:
+                raise ValueError('delete_missing_dataset must be true or false')
```

Comment:
Why did this change? If we're loading the config with `json.loads()` then the true/false values should be correctly converted to booleans.